src/value.rs

```diff
@@ -83,6 +81,10 @@ impl Value {
     }
 }
 
+pub fn escape_xml(s: &String) -> String {
```

Comment:
This could take a `&str` instead of a `&String` (that's the preferred way of passing immutable strings in Rust)